hugegraph-dist/src/assembly/static/bin/wait-storage.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Copyright 2023 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
```

Comment:
we also need add the refer of this file in root [LICENSE](https://github.com/apache/incubator-hugegraph/blob/869fc81811017b9780297cfaafb22d0328478afe/LICENSE#L214) file & dist [LICENSE](https://github.com/apache/incubator-hugegraph/blob/869fc81811017b9780297cfaafb22d0328478afe/hugegraph-dist/release-docs/LICENSE#L222) file